pallets/gear/src/lib.rs

```diff
@@ -165,10 +165,14 @@ pub mod pallet {
         #[pallet::constant]
         type Schedule: Get<Schedule<Self>>;
 
-        /// The maximum amount of messages that can be produced in single run.
+        /// The maximum amount of messages that can be produced in one message processing.
```

Comment:
NOTE: actually is not limit  for single run, but for message all execution, because `outgoing` map is not cleaned between executions